bin/.gitattributes

```diff
@@ -0,0 +1 @@
+*.sh eol=lf
```

Comment:
I think we should not have `lf` here, but `auto`.
We still support windows to build ZooKeeper, we should be able to edit shell files there too.

Reply:
I don't think that's the issue here, but when I rechecked today, the line endings in the archive seem to match the respective OS's conventions. Probably the issue is gone, nobody else complained anyway.